src/app/data/forms/textfield.ts

```diff
@@ -3,6 +3,7 @@ import { XmlClass } from '../../decorators/xml.decorators';
 
 @XmlClass('textfield')
 export class Textfield extends FormControl {
+  value = undefined;
```

Comment:
Da alle FormControls ein value haben, wäre es am besten, das in die abstrakte Basisklasse aufzunehmen, anstatt in jedem einzelnen FormControl